File: app/src/main/java/com/hippo/ehviewer/widget/MyEasyRecyclerView.java
package com.hippo.ehviewer.widget;

import android.annotation.SuppressLint;
import android.content.Context;
import android.util.AttributeSet;
import android.util.Log;
import android.view.MotionEvent;

import com.hippo.easyrecyclerview.EasyRecyclerView;

public class MyEasyRecyclerView extends EasyRecyclerView {
    public MyEasyRecyclerView(Context context) {
        super(context);
    }

    public MyEasyRecyclerView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public MyEasyRecyclerView(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    @SuppressLint("ClickableViewAccessibility")
    @Override
    public boolean onTouchEvent(MotionEvent ev) {
        try{
            return super.onTouchEvent(ev);
        }catch (IllegalArgumentException e){
            Log.e("RecyclerView", "IllegalArgumentException",e);
        }
        return  true;
    }

}

File: app/src/main/java/com/hippo/ehviewer/widget/MyEasyRecyclerView.kt
package com.hippo.ehviewer.widget

import android.annotation.SuppressLint
import android.content.Context
import android.util.AttributeSet
import android.util.Log
import android.view.MotionEvent
import com.hippo.easyrecyclerview.EasyRecyclerView

class MyEasyRecyclerView : EasyRecyclerView {
    constructor(context: Context) : super(context)

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)

    constructor(context: Context, attrs: AttributeSet?, defStyle: Int) : super(
        context,
        attrs,
        defStyle
    )

    @SuppressLint("ClickableViewAccessibility")
    override fun onTouchEvent(ev: MotionEvent?): Boolean {
        try {
            return super.onTouchEvent(ev)
        } catch (e: IllegalArgumentException) {
            Log.e("RecyclerView", "IllegalArgumentException", e)
        }
        return true
    }
}
